src/audio/dai.c

```diff
@@ -837,9 +837,11 @@ static int dai_copy(struct comp_dev *dev)
 		 dev->direction, copy_bytes,
 		 samples / buf->stream.channels);
 
-	/* return if it's not stream start */
-	if (!copy_bytes && dd->start_position != dev->position)
+	/* return if nothing to copy */
+	if (!copy_bytes) {
+		comp_warn(dev, "dai_copy(): nothing to copy");
 		return 0;
+	}
 
```

Comment:
Care to elaborate on why this change is needed. I mean, is there a specific usecase where you see an impact without this change?

I want to better understand the issue and also to have it written in the commit message. I guess, this is related to codec feature.

Reply:
it's just to record this information about missing dai_copy in logs.

Reply:
@mrajwa ok, i see. thanks!